GDSCommon-Demo/GDSCommon-Demo/Mocks/MockQRScanningViewModel.swift

```diff
@@ -4,18 +4,22 @@ import UIKit
 class MockQRScanningViewModel: QRScanningViewModel {
     let title: String
     let instructionText: String
+    let rightBarButtonTitle: GDSLocalisedString? = "right bar button"
     
```

Comment:
Very small nit: should the 'r' be 'R' just to stay consistent with the other views with a rightBarButton?

Reply:
I don't think any of the rightBarButtonTitle's are consistent on the mocks currently 😅